deposit/src/main/java/edu/unc/lib/deposit/work/DepositSupervisor.java

```diff
@@ -187,7 +187,7 @@ public DepositSupervisor() {
     }
 
     private static enum Queue {
-        PREPARE, DELAYED_PREPARE, CDRMETSCONVERT, PREPARE_HIGH_PRIORITY;
+        PREPARE, DELAYED_PREPARE, CDRMETSCONVERT, PREPARE_HIGH_PRIORITY, PREPARE_MIGRATION;
```

Comment:
Sorry to be a pain, but since you've gone with the `low` priority thing, I'd suggest naming the queue `PREPARE_LOW_PRIORITY` as well